UIKitDZ/UIKitDZ/Extensions/UIViewController+Extension.swift

```diff
@@ -0,0 +1,89 @@
+// UIViewController+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import UIKit
```

Comment:
```suggestion
import UIKit
```
фаундэйшн можно убрать